include/api/CAnomalyJobConfig.h

```diff
@@ -248,9 +248,11 @@ class API_EXPORT CAnomalyJobConfig {
 
         void parse(const rapidjson::Value& json);
 
-        bool updateFilters(const boost::property_tree::ptree& propTree);
+        void parse();
```

Comment:
Please add Doxygen comments to this method and the one immediately above saying why they do and why they're different.